public/assets/css/modules/button--generate-style-guide.css

```diff
@@ -0,0 +1,16 @@
+.button {
+  background-color: #C42872;
+  color: #fff;
+  height: 113px;
+  position: absolute;
+  right: -56px;
+  top: 300px;
+  transform: rotateZ(45deg) rotateX(0deg) rotateY(180deg);
+  z-index: 10;
+  width: 113px;
+}
+
+#text-button {
```

Comment:
First, We tried to use a class, but we changed this for ID because it took the classes of bootstrap over our classes, then we decided to use ID to give it more priority.